Update portal.json

alexrodba opened the pull request:
Corrected typos and style. Added missing translations. Added coherent style (tu vs vostè) across files.

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

alexrodba committed:
Update portal.json

Corrected typos and style. Added missing translations. Added coherent style (tu vs vostè) across files.

cathysarisky commented on ghost/i18n/locales/ca/portal.json:
```diff
@@ -46,166 +46,166 @@
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "En cas que es rebi un error permanent en intentar enviar un butlletí, els correus electrònics es desactivaran en aquest compte.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Afegeix {{senderEmail}} a la llista de contactes del teu client de correu electrònic. Això indica al teu proveïdor de correu que els correus electrònics enviats des d'aquesta adreça haurien de ser de confiança.",
+    "Invalid email address": "Adreça de correu electrònic invàlida",
+    "Jamie Larson": "Jamie Larson",
```
Jamie Larson is intended to be a placeholder name.  Feel free to suggest a name that would be recognizable to Catalan speakers.

alexrodba replied:
Awesome, will do it.

cathysarisky commented:
Hi @Zagur ! Do you have a moment to review these Catalan refinements?

alexrodba committed:
Update portal.json

Added more cohesiveness between imperative vs infinitive in buttons + other small style and typos fixes.

alexrodba reviewed (commented):
Added further changes

alexrodba committed:
Merge branch 'main' into patch-4

cathysarisky commented:
Hi @alexrodba !  Hi @Hana-Gausfain !  Could you two please discuss/look over each others' PRs, as the current Catalan contributors? :)

Zagur commented:
> Hi @Zagur ! Do you have a moment to review these Catalan refinements?

I'm currently on vacation. I can review it next week without any problems.

9larsons committed:
Merge branch 'main' into patch-4

9larsons committed:
Merge branch 'main' into patch-4
